Automattic/woocommerce-payments
# Updated blocks checkout payment method label design

Fixes #9474
Fixes #9475

#### Changes proposed in this Pull Request

This PR adds the Test mode label and updates the Credit Card logo for the Payment Method labels on the Blocks checkout.

<img width="478" alt="image" src="https://github.com/user-attachments/assets/d22ca39f-2125-4c17-beb7-8f2bb2e8198b">

<img width="475" alt="image" src="https://github.com/user-attachments/assets/cb6691cf-3718-42fb-bdeb-13edc02a15d3">

<img width="474" alt="image" src="https://github.com/user-attachments/assets/1a327cb0-ef5c-4c33-afad-de3d4cb9925b">


#### Testing instructions

- Go to the Blocks Checkout page and ensure it is rendered as expected

-------------------


**Post merge**
